lib/matplotlib/tri/tritools.py

```diff
@@ -61,22 +54,21 @@ def circle_ratios(self, rescale=True):
 
         To avoid unduly low values due to a difference of scale between the 2
         axis, the triangular mesh can first be rescaled to fit inside a unit
-        square with :attr:`scale_factors` (Only if *rescale* is True, which is
+        square with `scale_factors` (Only if *rescale* is True, which is
         its default value).
 
         Parameters
         ----------
         rescale : bool, default: True
             If True, a rescaling will be internally performed (based on
-            :attr:`scale_factors`, so that the (unmasked) triangles fit
```

Comment:
same

Reply:
same (It's actually a property and now links there.)